source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,263 @@
 ---
 title: HTML coding style
-last_reviewed_on: 2016-02-03
+last_reviewed_on: 2019-05-24
 review_in: 6 months
 ---
 
 # <%= current_page.data.title %>
 
-## HTML5
+## Progressive enhancement
 
-* Always quote attribute values - the decision tree for remember when to
-  selectively quote is too much mental overhead, so just quote them
-* Always use well-formed HTML - implicitly closed elements such as
-  `<p>` cause unnecessary parsing overhead and can cause issues in
-  accessibility
+HTML should be written in such a way that conforms with our [browser support model](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in).
 
-## Use [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) roles appropriately
+For example, Internet Explorer 8 doesn't support the `<video>` element. If you
+need to support IE8, make sure that an appropriate fallback is in place:
 
-### Guidance on using [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) landmarks
+```html
+<video width="320" height="240" controls>
+  <source src="movie.mp4" type="video/mp4">
+  <source src="movie.ogg" type="video/ogg">
+  Your browser does not support the video tag.
+</video>
```

Comment:
Fair enough 👍this was written when I was under the impression we supported IE8, as per https://github.com/alphagov/govuk-frontend#browser-support

I now know that's out of date.

Reply:
I don't know if it's out of date or not - best check on the Slack channel - but that looks like the browsers support by GOV.UK Frontend, rather than the browsers that the [Service Manual requires a service to support](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in).

Reply:
Indeed, GOV.UK Frontend has stricter standards for those teams that still need to support IE8

Reply:
In that case I think it's worth linking to both browser support tables, otherwise it sounds like we have one global set of supported browsers when really we do not. I've reworded this section 👍

Reply:
A proper fallback could be a link to the video so a user could download it and open in a video player of their choice.